python/hail/matrixtable.py

```diff
@@ -2725,12 +2725,21 @@ def from_rows_table(cls, table: Table) -> 'MatrixTable':
         ----------
         table : :class:`.Table`
             The table to be converted.
+        partition_key : :obj:`str` or :obj:`list` of :obj:`str`
+            Partition key(s).
 
         Returns
         -------
         :class:`.MatrixTable`
         """
-        jmt = scala_object(Env.hail().variant, 'MatrixTable').fromRowsTable(table._jt)
+        if partition_key is not None:
+            if isinstance(partition_key, str):
+                partition_key = [partition_key]
+            if len(partition_key) == 0:
+                raise ValueError('partition_key may not be empty')
+            elif list(table.key)[:len(partition_key)] != partition_key:
+                raise ValueError('partition_key must be a prefix of table key')
```

Comment:
Yet the docs say the table must be keyed by a single field? Also, you should document such requirements.

Reply:
This is an experimental method that will be removed as soon as we have partitioned tables (a few weeks probably). Is it worth the hassle? I'll make these changes if you think so

Reply:
I still think the docs should reflect the current behavior if doing so is not onerous. Here you just need to change/add a few words.